test/utils/test_negative_sampling.py

```diff
@@ -153,16 +159,72 @@ def test_structured_negative_sampling():
     assert (adj & neg_adj).sum() == 0
 
     # Test with no self-loops:
-    edge_index = torch.LongTensor([[0, 0, 1, 1, 2], [1, 2, 0, 2, 1]])
+    #edge_index = torch.LongTensor([[0, 0, 1, 1, 2], [1, 2, 0, 2, 1]])
```

Comment:
```suggestion
```

Reply:
@danielecastellana22 this line was causing the lint issue. But why was it commented out? was that just a temporary change?

Reply:
No, it can be removed. 
I think that in the previous test, there were two different `edge_index` to make sure that the structured negative sampling was feasible with and without the negative self-loops. 

Since I changed the structured negative sampling, now it is feasible also with the first definition of `edge_index`